pom.xml

```diff
@@ -817,6 +815,7 @@
       <id>jenkins-release</id>
       <properties>
         <skipTests>${release.skipTests}</skipTests>
+        <findbugs.skip>${release.skipTests}</findbugs.skip>
```

Comment:
unrelated but LGTM

Reply:
Yeah could be filed separately; just noticed that deployment was unnecessarily slow.